learn/os_setup/install_gfortran.md

```diff
@@ -12,17 +12,17 @@ GFortran is the name of the [GNU Fortran project](https://gcc.gnu.org/fortran/).
 Three sources provide quick and easy way to install GFortran compiler on Windows:
 1. [http://www.equation.com](http://www.equation.com/servlet/equation.cmd?fa=fortran), provides 32 and 64-bit x86
  executables of the latest (10.2) gcc-version.
-￼2. [TDM GCC](https://jmeubank.github.io/tdm-gcc/articles/2020-03/9.2.0-release), provides 32 and 64-bit x86 executables of the 9.2 gcc-version.
-￼3. [Mingw-w64](http://mingw-w64.org/doku.php/download/mingw-builds) provides a 64-bit x86 executable of the 7.2 gcc-version.
+2. [TDM GCC](https://jmeubank.github.io/tdm-gcc/articles/2020-03/9.2.0-release), provides 32 and 64-bit x86 executables of the 9.2 gcc-version.
+3. [Mingw-w64](http://mingw-w64.org/doku.php/download/mingw-builds) provides a 64-bit x86 executable of the 7.2 gcc-version.
```

Comment:
I suggest: [MinGW-w64/MSYS2](https://github.com/msys2) provides 32 and 64-bit x86 executable of the 10.2 gcc-version